scoap3/cli_fixes.py

```diff
@@ -304,3 +304,36 @@ def change_oai_hostname(dry_run, ids, old_hostname, new_hostname):
 
     if dry_run:
         error('NO CHANGES were committed to the database, because --dry-run flag was present.')
+
+
+@fixdb.command()
+@with_appcontext
+@click.option('--dry-run', is_flag=True, default=False,
+              help='If set to True no changes will be committed to the database.')
+@click.option('--ids', default=None, help="Comma separated list of recids to be processed. eg. '98,324'.")
```

Comment:
default for `ids` should be `[]`

Reply:
That's true, it would be nicer. In fact best would be an empty tuple `()`, but actually `None` will be handled as well properly.